jsmime.js

```diff
@@ -1,7 +1,5 @@
-define(function(require) {
```

Comment:
I'm not changing this to node-only style module syntax. If the syntax is to change, it's more likely to be UMD module style or possibly ES6 modules. (I've mostly been holding off in the hopes that ES6 modules will come to save us all).